### ISSUE TITLE
Long author list is truncated

### ISSUE THREAD

**Issue by akira-okumura** [r-xue/ads2bibdesk#23]
```
ads2bibdesk 2023APh...15002850A
```
The above command imports the paper information as expected but its author list is truncated and `and et al.` is added. Changing the ADS code to DOI `10.1016/j.astropartphys.2023.102850` does not help.

**Comment by mkounkel** [comment]
Does it make a difference though? When citing, journals truncate it to just first 3 authors anyway...

**Comment by r-xue** [comment]
If you use `bibtexabs` format [here](https://github.com/r-xue/ads2bibdesk/blob/c10668c7c8dfc8c7f7c7eb6de60c778e79dafd0f/ads2bibdesk/ads2bibdesk.py#L183), you can get the full list in the exported bibtex entry. However, as I noted in the comment there, it needs the newer version of [ads](https://github.com/andycasey/ads), which has not been pushed to PyPI (its entry there has not been updated for years).

In short, I likely won't change anything for now since it needs a dependency update first. But if you really need the fix, you can hack it locally: get the latest `ads` and flip the line mentioned above to `use_bibtexabs=True`.

**Comment by akira-okumura** [comment]
> Does it make a difference though? When citing, journals truncate it to just first 3 authors anyway...

Yes, it does. If long author lists are truncated, I cannot find my name from the BibDesk search field. Having 100 to 1000 authors is quite common in my filed.

**Comment by akira-okumura** [comment]
> `use_bibtexabs=True`.

Thank you very much. It works.

**Comment by r-xue** [comment]
> Yes, it does. If long author lists are truncated, I cannot find my name from the BibDesk search field. Having 100 to 1000 authors is quite common in my filed.

I see. This does sound like a legitimate reason to get it fixed.
